eslint.config.js

```diff
@@ -33,7 +33,8 @@ module.exports = [
       '@typescript-eslint/ban-types': 'off',
       '@typescript-eslint/no-unused-vars': 'error',
       '@typescript-eslint/no-explicit-any': 'error',
-      '@typescript-eslint/no-non-null-assertion': 'error'
+      '@typescript-eslint/no-non-null-assertion': 'error',
+      '@typescript-eslint/no-require-imports': 'off'
```

Comment:
We use imports or requires based on the file (test files, examples, eslint config itself, postinstall, etc.). Instead of having to disable it or ignore it for every file we want, easier to just disable the rule and let us decide when to use require or import per file.